app/src/main/java/sdp/moneyrun/ui/game/EndGameActivity.java

```diff
@@ -123,6 +124,7 @@ public void updateUser(@NonNull String playerId, int gameScore) {
         pdp.getUserTask(playerId).addOnCompleteListener(task -> {
             if (task.isSuccessful()) {
                 User p = pdp.getUserFromTask(task);
+                System.out.println(p);
```

Comment:
You should remove this, or change it to a log

Reply:
done